### PR TITLE
[doc] More info for pulling tags for Docker beginners.

### PR DESCRIPTION
**Pull request by 130s**
Information to help Docker beginners (like #15, which is never clear from the official `hub` page https://hub.docker.com/r/rosindustrial/core).

I don't know if I should just update `core` section or somwhere top level.

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
Perhaps this should moved to the top, because it applies to all the Docker repositories.
I might make sense to include a link to the tags for each repo.

**Comment by 130s** [comment]
Updated please review @ipa-mdl

**Comment by 130s** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -1,34 +1,41 @@
 # docker
 ROS-Industrial container tools.
 
-## caffe-cpu
+To pull images, you need to either specify `tag` or pull all tags with `-a` option. Taking `core` repo as an exmaple,
```
Comment (mathias-luedtke) [inline README.md]: exmaple -> example

**Review by mathias-luedtke — COMMENTED** [review]
If I am not mistaken, die nvidia and gpu images must be run with https://github.com/NVIDIA/nvidia-docker

**Comment by AustinDeric** [comment]
What about reference the Docker documentation and assuming the user is familiar with it.  That would make this repo simpler and easier to maintain.  With that said i do think the examples and documentation needs to be improved for this repo.  I was thinking about adding github pages.

Everything else in this PR is great and i would love to merge it!

**Comment by AustinDeric** [comment]
@130s I went ahead and used github's resolve feature for the merge conflicts, i have never used it before and it was really easy to use.  I removed the info about pulling the images, i think we should definitely have examples of how to use these though!  Maybe github pages with examples?  i'd be willing to help

**Pull request merged**